boards/arm64/nxp_ls1046ardb/nxp_ls1046ardb_defconfig

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Platform Configuration
+CONFIG_SOC_SERIES_LS1046A=y
+CONFIG_SOC_LS1046A=y
+CONFIG_BOARD_NXP_LS1046ARDB=y
+CONFIG_ARM64_VA_BITS_40=y
+CONFIG_ARM64_PA_BITS_40=y
+# 1.8GHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=1800000000
+
+# Zephyr Kernel Configuration
+CONFIG_SRAM_BASE_ADDRESS=0xc0000000
+CONFIG_SRAM_SIZE=1024
```

Comment:
This should be get directly from the DT, no need to have this here.

Reply:
It is a good idea, but I found CONFIG_SRAM_SIZE is also used in include/arch/arm64/scripts/linker.ld

 48 #define RAM_SIZE (CONFIG_SRAM_SIZE * 1K)
 49 #define RAM_ADDR CONFIG_SRAM_BASE_ADDRESS

so any idea how we can assign the value to these macro by getting informaton from DT? thanks.

Jiafei.

Reply:
The default value for `CONFIG_SRAM_SIZE` (and also `SRAM_BASE_ADDRESS`) is taken directly from the DT, see:

https://github.com/zephyrproject-rtos/zephyr/blob/8cae4661003525102be36767d97219b00b3697f3/arch/Kconfig#L161-L163

As you can see the default value is taken directly from the DT using the `zephyr,sram` handle.

https://github.com/zephyrproject-rtos/zephyr/blob/8cae4661003525102be36767d97219b00b3697f3/arch/Kconfig#L159

Reply:
Fine, will update the patch, thanks.